content/developer/cli.rst

```diff
@@ -316,11 +318,12 @@ Emails
 
 .. option:: --smtp-ssl-certificate-filename <path/to/cert.pem>
 
-    SSL certificate used for SMTP authentication
+    An SSL certificate is to be used for authentication. If set, then "smtp-ssl-private-key" is
+    required.
 
 .. option:: --smtp-ssl-private-key-filename <path/to/key.pem>
 
-    SSL private key used for SMTP authentication
+    An SSL private key is used for authentication. If set, then "smtp-ssl-certificate" is required.
```

Comment:
Would add backticks on the parameter, not quotes, since it's a code-related item.
```suggestion
    An SSL private key is used for authentication. If set, then `smtp-ssl-certificate` is required.
```